azure-quantum/tests/unit/recording_updater.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/env python
+# -*- coding: utf-8 -*-
+##
+# recording_updater.py: Updates test recordings with dummy values
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+
+## IMPORTS ##
+
+import glob
+import os
+import re
+
+
+class RecordingUpdater:
+    """RecordingUpdater
+    
+    Replaces sensitive data with dummy values in network recordings.
+    """
+    dummy_uid = "00000000-0000-0000-0000-000000000000"
+    dummy_rg = "dummy-rg"
+    dummy_ws = "dummy-ws"
+
+    def __init__(self, recording_directory, recording_glob):
```

Comment:
Docstrings and type hints would be useful for these methods, it's not strictly required for internal methods though but for the constructor and `update_recordings_with_dummy_values` would be helpful.